modules/core/test/v2/integration/coins/xrp.ts

```diff
@@ -28,7 +28,8 @@ describe('XRP:', function() {
     nock.cleanAll();
   });
 
-  it('Should generate wallet with custom root address', function() {
+  // TODO enable when initialization txs are built in platform
```

Comment:
Add ticket number to the `TODO(BG-20879):`

Reply:
done